cypress/integration/drag-svg.ts

```diff
@@ -154,8 +154,16 @@ describe("Drag SVG & Layout", () => {
     it("Drags the element by the defined distance", () => {
         cy.visit("?test=drag-svg&layout=true")
             .get("[data-testid='draggable']")
-            .wait(200)
-            .trigger("pointerdown", 0, 0, { force: true })
+            .wait(50)
+            .should(($draggable: any) => {
+                const draggable = $draggable[0] as HTMLDivElement
+                const { left, top } = draggable.getBoundingClientRect()
+
+                expect(left).to.equal(30)
+                expect(top).to.equal(30)
+            })
+            .wait(50)
+            .trigger("pointerdown", 50, 50, { force: true })
```

Comment:
Did this test just change significantly which is why all the numbers changed?

Reply:
So I've manually verified that these tests are working the same but it's more that we've changed that the pointer's position on an element gets calculated on pointer down, rather than when the pan initiates, the avoid pointer slippage.